k8s/bases/swe-d2cc/swe-d2cc-minio-automatic-purge.yaml

```diff
@@ -0,0 +1,44 @@
+kind: CronJob
+apiVersion: batch/v1
+metadata:
+  name: swe-d2cc-boundary-minio-automatic-purge
+  labels:
+    app: gridcapa
+    component: boundary-minio-automatic-purge
+    region: swe
+    process: d2cc
+spec:
+  schedule: "0 0 * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 3
+  successfulJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: swe-d2cc-boundary-minio-automatic-purge
```

Comment:
Could you please rename the container swe-d2cc-minio-automatic-purge to avoid confusion?